board/milkv/duo/overlay/etc/init.d/S05runonce

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+#
+# Execute scripts, that are intended to run only once
+#
+case "$1" in
+  start)
+
+        for file in /etc/runonce.d/*
+        do
+                if [ ! -f "$file" ]
+                then
+                        continue
+                fi
+                
+                if "$file" >> /tmp/runonce.log 2>&1
+                then
+                        mv "$file" "/etc/runonce.d/ran/"
```

Comment:
Hello~ What about adding a `sync` here after we move the file, to make sure the file has been indeed written back to the filesystem?

Reply:
It make sense, as a bonus `sync` will also commit writes (if any) made by called script. Let me add this.